blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+"""This module contains a MindMeld application"""
+from mindmeld import Application
+from mindmeld.models import entity_features
+from mindmeld.markup import process_markup
+from mindmeld.core import FormEntity
+import random
+import json
+
+app = Application(__name__)
+
+__all__ = ['app']
+
+user_data = {}
+
+sample_users = {
+    "johndoe123" : 0,
+    "larry_l12" : 1,
+    "splashbro30" : 2,
+}
+
+def pull_data(request):
```

Comment:
Make these functions private if they're only needed by this file?
```suggestion
def _pull_data(request):
```